README.md

```diff
@@ -200,3 +200,55 @@ From the _root_ directory of the repository:
 ```shell
 pytest tests/
 ```
+
+## Docker Setup:  
+This Section deals with setting up of Variation Normalizer via docker. 
+
+The Variation Normalizer depends upon several Modules , therefore its recommended to setup docker containers of these Modules before starting the Variation Normalizer container.Please ensure the target machine( where the Variation Normalizer is to be deployed has docker installed, otherwise docker commands wont work.  
+To Create Docker network , Please type following command.  
+command : docker network create <"name of the network> for e.g we have used "tulip-net"  
+Please follow below steps for Docker Setup of Variation Normalizer and its dependant containers.  
+
+## SeqRepo  
+Variation Normalizer depends on SeqRepo database. We need to create docker image for Seqrepo. It is recomended to start first with this image as volume attached to Seqrepo takes time to download and its size depending upon the version is 10 GB +.  
```

Comment:
```suggestion
Variation Normalizer depends on [Biocommons SeqRepo](https://github.com/biocommons/biocommons.seqrepo).  It is recommended to have the image as a volume attached to SeqRepo since the size exceeds 10 GB and can take a while to download.
```